### PR TITLE
Display seed in the CLI output.

### LINKED ISSUE #267: Display seed in the CLI output
It will help to simplify errors reproducing

### PR DESCRIPTION
**Pull request by svtkachenko**
Closes #267 

Output example:
```
=== ERRORS ===
____ GET: /api/flaky ____
hypothesis.errors.Flaky: Tests on this endpoint produce unreliable results: 
Falsified on the first call but did not on a subsequent one

Used seed       : 246442565675977375928223543448485731889
Query           : {'id': 0}

Run this Python code to reproduce this failure: 

requests.get('http://127.0.0.1:8081/api/flaky', params={'id': 0})
=== FAILURES ===
___ GET: /api/failure ____
Received a response with 5xx status code: 500

Used seed       : 313022266096038506880814748301664910156
Check           : not_a_server_error

Run this Python code to reproduce this failure: 

requests.get('http://127.0.0.1:8081/api/failure')
```

### CODE REVIEW

File: src/schemathesis/models.py [hunk 1]
```diff
@@ -147,6 +147,7 @@ class TestResult:
     checks: List[Check] = attr.ib(factory=list)  # pragma: no mutate
     errors: List[Tuple[Exception, Optional[Case]]] = attr.ib(factory=list)  # pragma: no mutate
     is_errored: bool = attr.ib(default=False)  # pragma: no mutate
+    seed: Optional[int] = attr.ib(default=None)  # pragma: no mutate
```
Comment (Stranger6667) [inline src/schemathesis/models.py]: I am not sure if it should be optional since there is always some seed in hypothesis

Comment (svtkachenko) [reply src/schemathesis/models.py]: If there is a hypothesis test. In case of invalid schema there is no test.

Comment (Stranger6667) [reply src/schemathesis/models.py]: aha, got it

File: src/schemathesis/runner/__init__.py [hunk 2]
```diff
@@ -164,6 +164,7 @@ def execute_from_schema(
             except Exception as error:
                 status = Status.error
                 result.add_error(error)
+            result.seed = getattr(test, "_hypothesis_internal_use_generated_seed", None)
```
Comment (Stranger6667) [inline src/schemathesis/runner/__init__.py]: It will be better to set it when `result` is created

Comment (svtkachenko) [reply src/schemathesis/runner/__init__.py]: `test._hypothesis_internal_use_generated_seed` does not exist at that moment. Hypothesis creates is during test execution

Comment (Stranger6667) [reply src/schemathesis/runner/__init__.py]: Got it, could you add a comment here?

**Pull request merged**